modules/cli/apply/apply.go

```diff
@@ -134,7 +135,7 @@ func readBundle(ctx context.Context, name, baseDir string, opts *Options) (*bund
 func createName(name, baseDir string) string {
 	path := strings.ToLower(filepath.Join(name, baseDir))
 	path = disallowedChars.ReplaceAllString(path, "-")
-	return multiDash.ReplaceAllString(path, "-")
+	return name2.Limit(multiDash.ReplaceAllString(path, "-"), 63)
```

Comment:
Will this just cut off the rest? What is there's two files in that path that share the same 63 char prefix? We might have a collision.

Reply:
Bumping this @StrongMonkey if you have a chance

Reply:
it will cut off also with a hash. So in your case it will have two different hashes in the end. (foobarbrah1, foobarbrah2)
foobar-hash1
foobar-hash2

Reply:
For your reference 
```
func Limit(s string, count int) string {
	if len(s) < count {
		return s
	}
	return fmt.Sprintf("%s-%s", s[:count-6], Hex(s, 5))
}
```

Reply:
Ahhh I missed that. Gotcha.